k8s/upgrade/src/user_prompt.rs

```diff
@@ -10,7 +10,7 @@ pub const REBUILD_WARNING: &str =  "\nThe cluster is rebuilding replica of some
 pub const SINGLE_REPLICA_VOLUME_WARNING: &str =  "\nThe list below shows the single replica volumes in cluster.\nThese single replica volumes may not be accessible during upgrade.\nTo skip this validation, please retry  with '--skip-single-replica-volume-validation` flag.";
 
 /// Warning to users before doing an upgrade.
-pub const CORDONED_NODE_WARNING: &str =  "\nBelow are the list of cordoned nodes. Since there are some cordoned nodes, \nthe rebuild will happen on another nodes so please have enough available free space or else the rebuild will get stuck forever.";
+pub const CORDONED_NODE_WARNING: &str =  "\nBelow are the list of cordoned nodes. Since there are some cordoned nodes, \nthe rebuild will happen on another nodes and the rebuild may get stuck forever. \nTo skip this validation, please retry  with '--skip-cordoned-node-validation` flag.";
```

Comment:
Can you update the message here, similar to the modified message present in the other PR?